src/app/app.component.html

```diff
@@ -1,6 +1,7 @@
 <h1>
   Components App
 </h1>
+<app-mtb-select></app-mtb-select>
```

Comment:
Can you put your component at the bottom. That way the components will be in the order that their respective PRs were accepted. Thanks.

Reply:
Thanks, I believe I have it correct now.